imports/plugins/core/dashboard/client/components/ShopLogoUrls.js

```diff
@@ -30,8 +30,8 @@ const shopLogoUrlsSchema = new SimpleSchema({
   "shopLogoUrls": Object,
   "shopLogoUrls.primaryShopLogoUrl": {
     type: String,
-    regEx: SimpleSchema.RegEx.Url,
-    optional: true
+    regEx: `${SimpleSchema.RegEx.Url}|(^$)`, // a URL or an empty string
+    optional: true,
```

Comment:
Tested locally and works as advertised 🎉 Can we remove this trailing comma? Looks like that isn't the convention for the rest of the file.